src/main/kotlin/com/terraformation/backend/tracking/db/ObservationStore.kt

```diff
@@ -1745,7 +1730,7 @@ class ObservationStore(
     recalculateSurvivalRate(
         OBSERVED_PLOT_SPECIES_TOTALS,
         OBSERVED_PLOT_SPECIES_TOTALS.MONITORING_PLOT_ID.eq(monitoringPlotId),
-        PLOT_T0_DENSITIES.MONITORING_PLOT_ID.eq(monitoringPlotId),
+        UpdatePlotSpeciesHelper(monitoringPlotId),
```

Comment:
Could the other two arguments here be turned into properties or methods on the helper class?